handleData.js

```diff
@@ -91,6 +91,20 @@ Player.prototype.printAddressBook = function() {
 }
 
 Player.prototype.sendMessage = function(recipient, content) {
+
+    if (recipient.playerName in this.conversation) {
+        this.conversation[recipient.playerName].push(content);
+    } else {
+        this.conversation[recipient.playerName] = [content];
+    }
+
+    if (this.playerName in recipient.conversation) {
+        recipient.conversation[this.playerName].push(content);
+    } else {
+        recipient.conversation[this.playerName] = [content];
+    }
+
+    /* Old method for player lookup and conversation establishment:
```

Comment:
This is unnecessary to keep. Git history lets you look into the past so you can recover the code later :)